src/labkey/Domain.ts

```diff
@@ -408,6 +408,58 @@ export function save(config: SaveDomainOptions): XMLHttpRequest {
     });
 }
 
+export interface ValidateDomainNameExpressionOptions extends CreateDomainOptions {
+    includeNamePreview: boolean
+}
+/**
+ * Validate name expression(s) for domain design.
+ */
+export function validateNameExpressions(config: ValidateDomainNameExpressionOptions): XMLHttpRequest {
+
+    let options: ValidateDomainNameExpressionOptions = arguments.length > 1 ? {
```

Comment:
Is this configuration necessary? We only do this for cases where we are providing backwards compatibility for methods that formerly took multiple parameters and now take a configuration object. I would recommend removing this for these new methods.

Reply:
@XingY Thanks for adjusting. Looks good.